installers/win-installer/pcgen.nsi

```diff
@@ -255,6 +272,7 @@ Section Uninstall
 	Delete /REBOOTOK "$INSTDIR\${APPDIR}\pcgen-release-notes-${SIMPVER}.html"
 	Delete /REBOOTOK "$INSTDIR\${APPDIR}\pcgen.exe"
 	Delete /REBOOTOK "$INSTDIR\${APPDIR}\pcgen.sh"
+	Delete /REBOOTOK "$INSTDIR\${APPDIR}\pcgen.bat"
```

Comment:
This was being left behind causing the folder not to be removed during installation.